algopy_testing/src/algopy_testing/arc4.py

```diff
@@ -223,6 +271,123 @@ def __bool__(self) -> bool:
         return bool(self.native)
 
 
+_TDecimalPlaces = typing.TypeVar("_TDecimalPlaces", bound=int)
+_MAX_M_SIZE = 160
+
+
+class _UFixedNxMMeta(type(_ABIEncoded), typing.Generic[_TBitSize, _TDecimalPlaces]):  # type: ignore  # noqa: PGH003
+    __concrete__: dict[tuple[type[_TBitSize], type[_TDecimalPlaces]], type] = {}  # noqa: RUF012
```

Comment:
wrapping the type annotation in `typing.ClassVar` should remove the need to ignore RUF012

Reply:
It has been fixed in this commit 682a52594452b917366e4f3557ec65e49fb58dce.